### ISSUE TITLE
Docs: ECMAScript Private Fields Links to Release Notes

### ISSUE THREAD

**Issue by fishcharlie** [microsoft/TypeScript-Website#584]
**Page URL:** https://www.typescriptlang.org/docs/handbook/classes.html#ecmascript-private-fields

**Issue:** It says `Right now, the best documentation for these private fields is in the TypeScript 3.8 release notes`. I don't believe documentation that links to release notes is a best practice.

**Recommended Fix:** This section should be reworked to include all the relevant information about ECMAScript Private Fields instead of sending the user to the release notes.

**Browser:** Safari Version 13.1.1 (15609.2.9.1.2)

**Comment by orta** [comment]
I'm not sure I agree, the release notes are documentation too.

We give an overview of the feature, and then show you where to learn about more about some of the trade-offs elsewhere.

**Comment by fishcharlie** [comment]
@orta Fair point. However it is an extra click away to get the information you need. And the release notes page doesn't support dark mode. So overall it just feels like the experience could be better and more streamlined.

<img width="1658" alt="Screen Shot 2020-05-17 at 11 47 24 AM" src="https://user-images.githubusercontent.com/860375/82155757-5a5d3280-9834-11ea-977f-8a5fb4fa968a.png">

<img width="1658" alt="Screen Shot 2020-05-17 at 11 47 32 AM" src="https://user-images.githubusercontent.com/860375/82155761-621cd700-9834-11ea-841a-8cdbf6ac7ee1.png">

**Comment by orta** [comment]
Yeah, I think moving the link to not be the version on the microsoft blog and instead be the version in the handbook will definitely fix that (and I'd rather keep folks on the site so that they have an obvious next port of call)

**Comment by orta** [comment]
Dark code samples in dark mode is in 👍🏻
